src/main/resources/ai/privado/rulevalidator/schema/policies.json

```diff
@@ -0,0 +1,163 @@
+      "$id": "#root/policies",
+      "title": "Policies",
+      "type": "array",
+      "default": [],
+      "items":{
+        "$id": "#root/policies/items",
+        "title": "Items",
+        "type": "object",
+        "required": [
+          "id",
+          "description",
+          "action",
+          "dataFlow",
+          "repositories"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "$id": "#root/policies/items/id",
+            "title": "Id",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy.Deny.Sharing.ContactToSlack"
+            ],
+            "pattern": "^.*$"
+          },
+          "name": {
+            "$id": "#root/policies/items/description",
+            "title": "Name",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy to restrict Contact Information being shared with Slack"
+            ],
+            "pattern": "^.*$"
+          },
+          "description": {
+            "$id": "#root/policies/items/description",
+            "title": "Description",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy to restrict Contact Information being shared with Slack"
+            ],
+            "pattern": "^.*$"
+          },
+          "action": {
+            "$id": "#root/policies/items/action",
+            "title": "Action",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Deny"
+            ],
+            "pattern": "(Allow|Deny)"
+          },
+          "dataFlow": {
+            "$id": "#root/policies/items/dataFlow",
+            "title": "Dataflow",
+            "type": "object",
+            "required": [
+              "sources"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "sources": {
+                "$id": "#root/policies/items/dataFlow/sources",
+                "title": "Sources",
+                "type": "array",
+                "default": [],
+                "items":{
+                  "$id": "#root/policies/items/dataFlow/sources/items",
+                  "title": "Items",
+                  "type": "string",
+                  "format": "regex",
+                  "default": "",
+                  "examples": [
+                    "Data.Sensitive.ContactData.*"
+                  ],
+                  "pattern": "^.*$"
+                }
+              },
+              "sinks": {
+                "$id": "#root/policies/items/dataFlow/sinks",
+                "title": "Sinks",
+                "type": "array",
+                "default": [],
+                "items":{
+                  "$id": "#root/policies/items/dataFlow/sinks/items",
+                  "title": "Items",
+                  "type": "string",
+                  "format": "regex",
```

Comment:
`**` is allowed here

Reply:
@HiteshMah-Jan [open]

Reply:
The format attr. is the type/medium we will be validating values with. Meaning we would use a regex to validate values against repositories. The type of value for this key is set to string, The schema version does support validation of values for being a valid regex though, but I didn't use it.